components/create_comment/__snapshots__/create_comment.test.jsx.snap

```diff
@@ -712,28 +648,12 @@ exports[`components/CreateComment should match snapshot, empty comment 1`] = `
         values={Object {}}
       />
     }
-    message={
-      <FormattedMessage
-        defaultMessage="By using @all or @channel you are about to send notifications to {totalMembers} people. Are you sure you want to do this?"
-        id="notify_all.question"
-        values={
-          Object {
-            "totalMembers": 2,
-          }
-        }
-      />
-    }
+    message=""
     modalClass=""
     onCancel={[Function]}
     onConfirm={[Function]}
     show={false}
-    title={
-      <FormattedMessage
-        defaultMessage="Confirm sending notifications to entire channel"
-        id="notify_all.title.confirm"
-        values={Object {}}
-      />
-    }
+    title=""
```

Comment:
why are these emptied?

Reply:
This is because prior to my change, the title were always getting set regardless if the confirmation modal was being shown or not. In my change, I start them off as blank and only set them if needed. So they don't won't always have values associated to them unless we will display the confirmation modal.